pkg/api/api/projects.go

```diff
@@ -0,0 +1,54 @@
+package api
+
+import (
+	"context"
+
+	"connectrpc.com/connect"
+	projectsv1alpha1 "go.jetpack.io/pkg/api/gen/priv/projects/v1alpha1"
+	"go.jetpack.io/pkg/api/gen/priv/projects/v1alpha1/projectsv1alpha1connect"
+	"go.jetpack.io/pkg/id"
+)
+
+func (c *Client) ListProjects(
+	ctx context.Context,
+	orgID id.OrgID,
+) ([]*projectsv1alpha1.Project, error) {
+	memberResponse, err := projectsv1alpha1connect.NewProjectsServiceClient(
```

Comment:
Do you really want to create a new service client in each request?

Why not initialize these as fields of the `Client` struct on `NewClient`

Reply:
Sure, don't feel strongly.